lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/AzureBatchBootstrapREEFLauncher.java

```diff
@@ -129,6 +171,35 @@ private static Configuration generateConfigurationFromJobSubmissionParameters(fi
         .build();
   }
 
+  private static Set<String> getAzureBatchInBoundNatPoolBackendPorts(
```

Comment:
One last comment - it looks like the tabbing/spaces are off, can you correct this? Thanks!